packages/components/app/styles/components/tabs.scss

```diff
@@ -1,9 +1,69 @@
 //
 // TABS
 //
-// properties within each class are sorted alphabetically
-//
 
-.hds-tabs {
-  // outline: 2px dotted red;
+$indicator-height: 3px;
+
+.hds-tabs {}
+
+// Sub-components:
+// TODO: MOVE!
+.hds-tab-list {
+  font-family: var(--token-typography-body-200-font-family);
+  font-weight: var(--token-typography-font-weight-medium);
+  padding-bottom: ($indicator-height / 2);
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  white-space: nowrap;
+
+  // Un-nest after moving
+  .hds-tab-list__list {
+    border-bottom: 1px solid var(--token-color-border-primary);
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+}
+
+.hds-tab {
+  padding: 3px 6px;
+  position: relative; // temporary: for positioning temp indicator
+
+  // temporary indicator
+  &:hover:after {
+    background: var(--token-color-foreground-action);
+    border-radius: 3px;
```

Comment:
Visually it's exactly the same. I'm following the Figma in this case but I would normally set something like this to a much higher pixel value so that it would still have the same rounded appearance even if the height were to change.